### DISCUSSION TITLE
How to  configure the AUC FCs display LCDs .

### DISCUSSION THREAD

**Discussion by Terracotta-6** [artisan-roaster-scope/artisan#1693] [General]
![aa3af0181dcc973e7d970d65586b1af](https://github.com/user-attachments/assets/137b09ba-1452-47a1-918f-837e110fe29d)
![4524a714d77ea636a45e55a71215f03](https://github.com/user-attachments/assets/2bd4aaaf-b101-4388-a00f-d60474eb35b1)

**Comment by roasterdave** [comment]
When the AUC LCD option is checked in the Statistics dialog an LCD is added to the display as seen in your screenshot.  Right clicking on the LCD toggles between three different values.  "AUC" is the current value of the AUC per the configuration in the Statistics dialog.  ">>AUC" is the difference between the Target AUC set in the configuration and the current AUC value.  "AUC FCs" is the AUC value starting from FCs to the present.
